src/EditorFeatures/TestUtilities/Remote/InProcRemostHostClient.cs

```diff
@@ -34,7 +34,10 @@ internal sealed class InProcRemoteHostClient : RemoteHostClient
             // make sure connection is done right
             var current = $"VS ({Process.GetCurrentProcess().Id})";
             var telemetrySession = default(string);
-            var host = await instance._rpc.InvokeAsync<string>(nameof(IRemoteHostService.Connect), current, telemetrySession).ConfigureAwait(false);
+            var uiCultureLCIDE = 0;
+            var cultureLCID = 0;
+
+            var host = await instance._rpc.InvokeAsync<string>(nameof(IRemoteHostService.Connect), current, uiCultureLCIDE, cultureLCID, telemetrySession).ConfigureAwait(false);
```

Comment:
Can't you change the culture within a VS session?  How does that get propagated to the remote host?

Reply:
this is a mock for testing.

Reply:
ah. mis-understood you question. no you can't change culture within VS session. you need to re-open VS.